client/components/TestRun/TestRun.css

```diff
@@ -1,13 +1,166 @@
-#test-iframe {
-    width: 80vw;
-    height: 65vh;
+/* Test Navigator */
+button.test-navigator-toggle.hide {
+    text-decoration: none;
+    color: #0b60ab;
+    border: 0;
+    background-color: transparent;
+    padding: 0 0 0 1.25em;
+    position: relative;
 }
 
-.testrun__button--right {
+button.test-navigator-toggle.hide:before {
+    content: '<<';
     position: absolute;
-    right: 0;
+    left: 0;
+    font-size: 0.75em;
+    font-weight: 700;
+    top: 0.25em;
 }
 
-.testrun__button-toolbar--margin {
-    margin: 1em 0;
+span.test-navigator-toggle-container {
+    position: absolute;
+    left: -20px;
+}
+
+button.test-navigator-toggle.show {
+    width: 21px;
+    height: 21px;
+    border: 2px solid gray;
+    border-radius: 3px;
+    background: transparent;
+    position: relative;
+}
+
+button.test-navigator-toggle.show:before {
+    content: '';
+    position: absolute;
+    width: 13px;
+    height: 13px;
+    left: 2px;
+    top: 2px;
+    background: linear-gradient(
+        180deg,
+        rgba(128, 128, 128, 1) 0%,
+        rgba(128, 128, 128, 1) 20%,
+        rgba(255, 255, 255, 1) 20%,
+        rgba(255, 255, 255, 1) 40%,
+        rgba(128, 128, 128, 1) 40%,
+        rgba(128, 128, 128, 1) 60%,
+        rgba(255, 255, 255, 1) 60%,
+        rgba(255, 255, 255, 1) 80%,
+        rgba(128, 128, 128, 1) 80%,
+        rgba(128, 128, 128, 1) 100%
+    );
+}
+
+button.test-navigator-toggle.show:after {
+    content: '';
+    position: absolute;
+    width: 2px;
+    height: 15px;
+    left: 4px;
+    top: 2px;
+    background: white;
+}
+
+.show {
+    width: 2em;
+}
```

Comment:
hey @isaacdurazo , do you know why this is here? it's causing the toast to look like this: 
![problem-with-toast](https://user-images.githubusercontent.com/583831/98027150-10a3dc80-1dc1-11eb-9a1b-15a530c9d459.png)